snaps/features/data-storage.md

```diff
@@ -11,6 +11,13 @@ Use the [`snap_manageState`](../reference/snaps-api.md#snap_managestate) API met
 100 MB of data to the user's disk and retrieve it at will.
 We recommend using this method for storing data in a Snap long term.
 
+:::caution important
+You can install Snaps into each user's MetaMask instance.
```

Comment:
```suggestion
Snaps are installed in each user's MetaMask instance.
```

The user installs the Snap, not the Snap developer. I think passive voice is fine here because we are emphasizing where the Snap is installed.